MonoGame.Framework/Rectangle.cs

```diff
@@ -290,6 +290,16 @@ public bool Contains(Rectangle value)
         }
 
         /// <summary>
+        /// Gets whether or not the provided <see cref="Rectangle"/> lies within the bounds of this <see cref="Rectangle"/>.
+        /// </summary>
+        /// <param name="value">The <see cref="Rectangle"/> to check for inclusion in this <see cref="Rectangle"/>.</param>
+        /// <param name="result"><c>true</c> if the provided <see cref="Rectangle"/>'s bounds lie entirely inside this <see cref="Rectangle"/>; <c>false</c> otherwise. As an output parameter.</param>
+        public void Contains(ref Rectangle value,out bool result)
```

Comment:
I don't see the point to a `out bool` here.  It can simply return `bool` and be just as efficient.